lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -10,6 +10,9 @@ class InteractiveOptions
         # Prompts the user with options
         # Uses an interactive session to allow the user to pick an answer
         # Can use arrows, y/n, numbers (1/2), and vim bindings to control
+        # For more than 9 options, hitting 'e', ':', or 'G' will enter select
```

Comment:
What's the rationale behind `e`, `:`, and `G`?

Reply:
`e` just because it's the first letter of "enter".  `:` and `G` are both vim keybindings for going to a specific line.  I figured that, since vim keybinds are supported for up/down, you might want vim-style keys for other things as well.

Reply:
Alright, that seems fine. It threw me off because `G` in `less` is to go at the end of the file.
`:` and `/` are also used in `less` for similar meanings, so perhaps use those for the instructions? i.e. Keep all the current bindings, just change the instructions.

Reply:
In vim too. But for both, prefix it with a number and it goes there instead.

Reply:
@lavoiesl I'd prefer to err on the side of displaying a single, obvious keybinding and putting the others in the documentation.  I'm not against changing it, but I actually had no idea those keybindings existed for less and had to look them up for vim (I use Emacs, myself, but I doubt anyone wants to support those keybindings :stuck_out_tongue:).

Speaking of documentation, I should probably update the README with these new features.

Reply:
All good.